provider/core/src/hello_world.rs

```diff
@@ -128,10 +128,10 @@ impl DataProvider<HelloWorldV1Marker> for HelloWorldProvider {
             .get(langid)
             .map(|s| HelloWorldV1 { message: s.clone() })
             .ok_or_else(|| DataError::MissingResourceOptions(req.clone()))?;
+        let mut metadata = DataResponseMetadata::default();
```

Comment:
note (non blocking): Rust does support this syntax if you think it would be cleaner. I don't have a strong opinion either way.

```rust
DataResponseMetadata {
  data_langid: Some(langid.clone()),
  ..Default::default(),
}
```

the idiomaticness tradeoff is mostly that this doesn't need any `mut` which rust famously dislikes; but otoh it is a bit more unweildy-looking

Reply:
The nice syntax doesn't work for non_exhaustive sadly. https://github.com/rust-lang/rust/issues/70564